redshift_console/queries.yaml

```diff
@@ -24,13 +24,18 @@ query_alerts:
   AND query IN %s
   ORDER BY event_time ASC;
 table_load_errors:
-  SELECT stv_tbl_perm.name as table, colname as column, err_reason as error, starttime as time, COUNT(*) as errors_count
+  SELECT TRIM(pg_namespace.nspname) as schema,
+         TRIM(pg_class.relname) as table,
+         stl_load_errors.colname as column,
+         stl_load_errors.err_reason as error,
+         stl_load_errors.starttime as time,
+         COUNT(*) as errors_count
   FROM stl_load_errors
-  INNER JOIN (SELECT DISTINCT id,name FROM stv_tbl_perm) as stv_tbl_perm
-  ON stl_load_errors.tbl=stv_tbl_perm.id
+  INNER JOIN pg_class ON pg_class.OID=stl_load_errors.tbl
```

Comment:
How about we store the OID when we fetch the schema, and then in this query (and similar) we won't need to join with `pg_class` or `pg_namespace` (we will use our own lookup instead)?